### PR TITLE
Gitignore task: ignore wrong npm/yarn lock file #4

### PR DESCRIPTION
**Pull request by gigobyte**

### CODE REVIEW

File: packages/mrm-task-gitignore/index.js [hunk 1]
```diff
@@ -14,13 +15,15 @@ const ignores = [
 	'*.sublime-workspace',
 	'*.log',
 	'.eslintcache',
+	existsSync('package-lock.json') ? 'yarn.lock' : '',
```
Comment (sapegin) [inline packages/mrm-task-gitignore/index.js:18]: `existsSync('package-lock.json') && 'yarn.lock'` would be more clear probably.

**Review by sapegin — COMMENTED** [review]
Thanks! That was quick ;-)

File: packages/mrm-task-gitignore/index.js (hunk 1 above)
Comment (gigobyte) [reply packages/mrm-task-gitignore/index.js:18]: That would change the type of ignores to `(string | bool)[]`, I preferred to keep it `string[]`

Comment (sapegin) [reply packages/mrm-task-gitignore/index.js:18]: Using empty sting instead of an undefined or false is a hack anyway.

**Pull request closed without merging**